rust/cli/src/commands/update.rs

```diff
@@ -52,6 +55,45 @@ pub async fn run_update(args: UpdateArgs) -> Result<()> {
     Ok(())
 }
 
+fn ensure_in_vlayer_directory() -> Result<()> {
+    // Can happen if current working directory doesn't exist
+    let current_dir = std::env::current_dir()
+        .map_err(|e| Error::Upgrade(format!("Failed to get current directory: {e}")))?;
+
+    if check_current_dir_is_vlayer(&current_dir)? {
+        return Ok(());
+    }
+
+    if change_to_vlayer_subdir(&current_dir)? {
+        return Ok(());
+    }
+
+    Err(Error::Upgrade(
+        "vlayer update must be run from within a 'vlayer' directory, or from a directory containing a 'vlayer' subdirectory".to_string()
```

Comment:
```suggestion
        "vlayer update must be run from within <project root> or <project root>/vlayer/ directories".to_string()
```

I'd be very specific, because we also have `contracts/vlayer`.

Also, don't we have any other way of detecting being present in project root? The current solution is good enough, just wondering if there would be a more robust option.